src/Glpi/Search/Provider/SQLProvider.php

```diff
@@ -5140,7 +5140,7 @@ public static function giveItem(
                                                 $out .= \Search::LBBR;
                                             }
                                             $count_display++;
-                                            $out .= "<a href='mailto:" . $split2[1] . "'>" . $split2[1] . "</a>";
+                                            $out .= "<a href='mailto:" . $split2[1] . "'>" . \htmlspecialchars($split2[1]) . "</a>";
```

Comment:
Every value should be escaped in every piece of HTML code.

```suggestion
                                            $out .= "<a href='mailto:" . \htmlspecialchars($split2[1]) . "'>" . \htmlspecialchars($split2[1]) . "</a>";
```